lib/rules/no-invalid-link-text.js

```diff
@@ -99,7 +102,11 @@ export default class NoInvalidLinkText extends Rule {
         // In strict mode, LinkTo is ignored even with invalid text
         // This is because we can't tell if the component is actually a built-in LinkTo
         // or just has the same name. It's better to risk false negatives than false positives.
-        if (node.tag === 'a' || (node.tag === 'LinkTo' && !this.isStrictMode)) {
+        if (
+          node.tag === 'a' ||
+          (node.tag === 'LinkTo' && !this.isStrictMode) ||
+          this.config.linkComponents.includes(node.tag)
```

Comment:
shouldn't this also be exclusive to loose mode?

Reply:
not really, because we only extending functionality (non breaking) - user needs to manually specify component name.